tests/test_bin.py

```diff
@@ -0,0 +1,14 @@
+from roulette import Bin, Outcome
+from unittest import TestCase
```

Comment:
standard libs should come first in import order

Reply:
Done. Thanks.